checks/code_review.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Security Scorecard Authors
+// Copyright 2021 Security Scorecard Authors
```

Comment:
if this is changing, maybe 2022?

Reply:
Copyright dates do not need to be updated.
They only need to reflect current date if it's a net-new file.

Reply:
> Copyright dates do not need to be updated.
> They only need to reflect current date if it's a net-new file.

ref: https://github.com/ossf/scorecard/issues/1534 for discussion